src/openms/include/OpenMS/KERNEL/MRMTransitionGroup.h

```diff
@@ -335,6 +335,12 @@ namespace OpenMS
         }
       }
 
+      for (typename std::vector<SpectrumType>::iterator pr_it = precursor_chromatograms_.begin(); pr_it != precursor_chromatograms_.end(); ++pr_it)
```

Comment:
I think it goes beyond this PR but my idea was that we could store multiple isotopic traces and charge states as precursor signals ... maybe with their relative intensity? I am not sure about the naming yet ...